docs/source/rpc.rst

```diff
@@ -59,7 +62,7 @@ may send and receive gradient recording tensors over RPC. In the forward pass,
 we record when gradient recording tensors are sent over RPC and during the
 backward pass we use this information to perform a distributed backward pass
 using RPC. For more details see the design doc
-`here <https://github.com/pytorch/pytorch/pull/29175>`_.
+`here <https://pytorch.org/docs/master/notes/distributed_autograd.html>`_.
```

Comment:
nit: Is there a way to ensure this link points to `master` when the current doc is master and `stable` when the current doc is `stable`? For example, once this becomes stable clicking on this link would redirect users to the `master` doc instead of the stable doc.

Reply:
This should be the case in the latest update to this PR.